### ISSUE TITLE
It's time to make a new release both on GitHub and NPM

### ISSUE THREAD

**Issue by raphaklaus** [Marak/faker.js#621]
Some functionalities are not present in the last available release (such as credit card generation).

**Comment by wil93** [comment]
Same, I was just spending some time wondering why `faker.random.hexaDecimal(64)` was not working...

**Comment by MatthiasKunnen** [comment]
@Marak, is she still? This here https://github.com/nodejs/board states she is no longer since January although I might be misinterpreting this. If case you no longer want to maintain this library, can you give one or more contributors publishing rights?

**Comment by Marak** [comment]
I'm still seeing her name and photograph in that repository you've sent me as well as the main Node.js website.

https://imgur.com/a/Nol47ke

Once her information is removed from both the repository and Node.js website I will consider her involvement with Node.js leadership done.

**Comment by Marak** [comment]
To respond to your edit, we'll have to see what happens in July then.

**Comment by ezpuzz** [comment]
@Marak I believe the pages have been removed and restructured without her inclusion from my searching.

**Comment by Marak** [comment]
As of today, still waiting for https://github.com/nodejs/board/blob/master/README.md to be updated.

**Comment by mroderick** [comment]
@Marak https://github.com/nodejs/board/blob/master/README.md was updated yesterday

**Comment by MatthiasKunnen** [comment]
Correct, see PR https://github.com/nodejs/board/pull/79

**Comment by Marak** [comment]
It looks like Ashley Williams has now been removed completely from the
Node.js board and accompanying websites.

I'll see about pushing out a new release before the end of the month.

Let this be a warning for others who may consider allowing Ashley Williams
( or other similar non-coding entities ) to contribute or manage your
business or open source projects. Ashley was proudly a sexist and racist
figure who constantly berated my race and gender over Twitter and other
publicly viewed forums. Her group of friends and close supporters were even
more vile. They are the main reason I've switched my Twitter to private /
pretty much stopped using Twitter entirely.

There was no way I was going to continue to contribute or work with a
community who would trust such a horrible person in a leadership position.
